das/metrics.go

```diff
@@ -24,7 +25,10 @@ type metrics struct {
 	sampleTime    syncfloat64.Histogram
 	getHeaderTime syncfloat64.Histogram
 	newHead       syncint64.Counter
-	lastSampledTS int64
+	totalSampled  asyncint64.Gauge
+
+	lastSampledTS   int64
+	totalSampledInt int64
```

Comment:
Is it necessary to make it `int64`? Or we can also make it `uint64`

Reply:
I think `uint64` is more fitting

Reply:
although this will cause conversion when calling `.Observe` since the metric type is `AsyncInt64.Gauge`